tests/datalab/test_factory.py

```diff
@@ -13,15 +13,16 @@ def registry():
 def test_list_possible_issue_types(registry):
     issue_types = Datalab.list_possible_issue_types()
     assert isinstance(issue_types, list)
-    possible_issues = ["label", "outlier", "near_duplicate", "non_iid", "class_imbalance"]
+    possible_issues = ["outlier", "near_duplicate", "non_iid"]
```

Comment:
@elisno  I think it could be done in this way:
```
def list_possible_issue_types() -> List[str]:
        """... """
        return list(REGISTRY.keys()) + list(TASK_SPECIFIC_REGISTRY[self.datalab.task].keys())
```
or by mutating REGISTRY, but we want to avoid it.

Btw. I've started to test locally regression and noticed one small potential issue:
```
dl_regression = Datalab(data=df, label_name="target", task="regression")
datalab_regression.class_names
datalab_regression.get_info() 
```
This is problematic as for regression "number of classes" could be equal to number of datapoints, also for "extreme classification" - this could flood the screen ;) 
Sorry for spamming with commits, that should be squashed on my side.